build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -22,7 +22,7 @@ public class AzurePublisher
 
         public AzurePublisher()
         {
-            _connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING").Trim('"');
+            _connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING")?.Trim('"');
```

Comment:
I agree we should use EnvVars.EnsureVariable.

I placed the null operator before I made the other methods static. Will fix.